install_bootstrap

```diff
@@ -240,6 +241,8 @@ while [[ "${run_dockstore_launcher^^}" != 'Y' &&  "${run_dockstore_launcher^^}"
         ask_question "What is your Discourse URL (ends with a /)?" "$DISCOURSE_URL" "discourse_url" $discourse_url
         discourse_key='discourse_key'
         ask_question "What is your Discourse key?" "$DISCOURSE_KEY" "discourse_key" $discourse_key
+        discourse_category_id='discourse_category_id'
+        ask_question "What is your Discourse category id (9 for staging, 6 for production)?" "$DISCOURSE_CATEGORY_ID" "discourse_category_id" $discourse_category_id
```

Comment:
If these are the two choices, would it be simpler to tie this into "Are you deploying on production (true or false)?"

Reply:
Yes, though it could be the case that we change the category id we want to post to.